noman.el

```diff
@@ -32,33 +32,34 @@
 (require 'cl-lib)
 (require 'ansi-color)
 
-(defvar-local noman--last-command nil
-  "The last command that noman executed.")
+(defgroup noman nil "Command line help reader." :group 'applications :prefix "noman-")
 
-(defvar-local noman--buttons '()
-  "A list of buttons in the current noman buffer.")
+(defcustom noman-parsing-functions
```

Comment:
It's likely users (or other packages) will want to extend this, so it's a good candidate for a user option.